docs/setup-iptables.md

```diff
@@ -3,6 +3,11 @@
 In order to prevent the indy-node process from reaching of open file descriptors limit caused by clients connections it is strongly
 recommended to add iptables rule that limits the number of simultaneous clients connections for client port.
 
```

Comment:
I think a link of this doc and explicit statement about setting firewall rules needs to be part of `README.md` and `start-nodes.md`

Reply:
Updated _setup-iptables.md_ and _start-nodes.md_.